test/code-cache/test-code-cache-generator.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+// This test verifies that the binary is compiled with code cache and the
+// cache is used when built in modules are compiled.
+
+const common = require('../common');
+if (!common.enoughTestCpu || !common.enoughTestMem) {
```

Comment:
Is it that resource-intensive that it's need skipping?
Personally I don't mind less coverage, but I'm not sure this test actually that bad 🤷‍♂️ 
(we do have tests that process files 10s of MB in size, and take much longer then this one)

Reply:
This is testing something in tools, so I think we just need to make sure it runs fine for a start - if it comes back not enough, we can add more coverage, but we may not even settle down on a JS script in the end (for one I still prefer an executable like mkcodecache, although that may need much more refactoring to get implemented). The code cache functionality of v8 should already been tested somewhere else anyway.